README.adoc

```diff
@@ -244,12 +281,27 @@ EndpointIT.java
 include::finish/src/test/java/it/io/openliberty/guides/rest/EndpointIT.java[]
 ----
 
+// Static guide version
+ifndef::cloud-hosted[]
 You can test this service manually by starting a server and going to the 
 http://localhost:9080/system/properties-new[http://localhost:9080/system/properties-new^] URL. 
 However, automated tests are a much better approach because they trigger a failure if a change introduces a bug.
 JUnit and the JAX-RS Client API provide a simple environment to test the application. 
-You can write tests for the individual units of code outside of a running application server, or they
-can be written to call the application server directly. In this example, you will create a test that calls the application server directly.
+You can write tests for the individual units of code outside of a running application server,
```

Comment:
or they can be written to call the application server directly.
--->
or you can write them to call the application server directly.